src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -577,7 +586,7 @@ BOOST_AUTO_TEST_CASE(LogisticFunctionTest)
 
   CheckActivationCorrect<LogisticFunction>(activationData, desiredActivations);
   CheckDerivativeCorrect<LogisticFunction>(desiredActivations,
-      desiredDerivatives);
```

Comment:
I think this was correct before, and in the other places? I think you misunderstood the style guidelines.

Reply:
Thanks @favre49. Yup this was correct before too. And I am also aware of the 2 tab line wrapping, i thought having similar alignment will bring uniformity to the codebase.